src/App/App.scss

```diff
@@ -46,6 +46,13 @@ h4 {
     border-style: none none solid none;
     border-radius: 0;
   }
+  #remove-addon {
+    background-color: transparent;
+    color: $COLOR-dark-background;
+    border-color: $COLOR-dark-background;
+    border-style: none none solid none;
```

Comment:
If I click the delete button, there is a border / shadow appearing for a short time. This should not happen, therefore please check the button for :hover / :focus / :active css attributes.